src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1602,35 +1602,48 @@ function CategoricalArrays.categorical(df::AbstractDataFrame,
 end
 
 """
-    flatten(df::AbstractDataFrame, col::Union{Integer, Symbol})
-
-When column `col` of data frame `df` has iterable elements that define `length` (for example
-a `Vector` of `Vector`s), return a `DataFrame` where each element of `col` is flattened, meaning
-the column corresponding to `col` becomes a longer `Vector` where the original entries
-are concatenated. Elements of row `i` of `df` in columns other than `col` will be repeated
-according to the length of `df[i, col]`. Note that these elements are not copied,
-and thus if they are mutable changing them in the returned `DataFrame` will affect `df`.
+    flatten(df::AbstractDataFrame, cols::Union{ColumnIndex, AbstractVector, Regex, Not,
+            Between, All, Colon})
```

Comment:
Either keep this on the previous line, align them with the character after `Union{`, or add a line break before `cols` instead.

Reply:
I would break before `cols` - just as in the method definition.